src/routes/Routes.ts

```diff
@@ -0,0 +1,21 @@
+const Routes = {
+  ROOT: '/',
+  LOGIN: '/login',
+  SIGNUP: `/signup`,
+  HOME: '/home',
+  NOT_FOUND: '/not_found',
+  UNKNOWN_ROUTE: '*',
+};
+
+export type RoutesType = typeof Routes;
+export type RouteNames = keyof RoutesType;
+export type RoutePaths = RoutesType[RouteNames];
```

Comment:
**question:** Do we need to export those types right now?

Reply:
no need, but it will be good to have it, so I`ve let it here